Patch for a possible version 0.13

lhw opened the pull request:
Adds features and proposed changes from issue #4, #6, #7

lhw committed:
fixing issue #4,#6,#7

lhw committed:
Merge branch 'master' into issuesolving

lhw committed:
new availability field for users

lhw committed:
example

rrix commented:
Hi @lhw . We're finally catching up on SpaceAPI and trying to get ready for a 0.13 release of the API. I notice that at least some of this stuff is already pulled in to the 0.13rc1 that is represented in the current master branch HEAD (ie 6b8510cf1b391a8289bc35537554fcb292895acb). However, some of it is not! 

Also, 928168b confuses me, I'm not sure what it is supposed to do ...?

Can you please rebase this off of current master if you're still interested in making it happen? I'm going to be converting the documentation spec to markdown soon as #22 is merged, if that's easier to work with for you

lhw commented:
Hmm I can't really tell what 928168b was supposed to do as it breaks the actual json. I reverted that one. I also rebased on the latest master. But there werent any real changes anyhow.
